bin/stress-test/src/main.rs

```diff
@@ -0,0 +1,283 @@
+use std::{
+    path::{Path, PathBuf},
+    time::{Duration, Instant},
+};
+
+use anyhow::Context;
+use clap::{Parser, Subcommand};
+use miden_lib::{
+    account::{auth::RpoFalcon512, faucets::BasicFungibleFaucet, wallets::BasicWallet},
+    transaction::TransactionKernel,
+};
+use miden_node_block_producer::{
+    batch_builder::TransactionBatch, block_builder::BlockBuilder, store::StoreClient,
+    test_utils::MockProvenTxBuilder,
+};
+use miden_node_proto::generated::store::api_client::ApiClient;
+use miden_node_store::{config::StoreConfig, server::Store};
+use miden_objects::{
+    account::{AccountBuilder, AccountId, AccountStorageMode, AccountType},
+    asset::{Asset, FungibleAsset, TokenSymbol},
+    block::BlockHeader,
+    crypto::dsa::rpo_falcon512::{PublicKey, SecretKey},
+    note::{Note, NoteInclusionProof},
+    testing::note::NoteBuilder,
+    transaction::OutputNote,
+    Digest, Felt, MAX_OUTPUT_NOTES_PER_BATCH,
+};
+use miden_processor::crypto::{MerklePath, RpoRandomCoin};
+use rand::Rng;
+use rayon::iter::{IndexedParallelIterator, IntoParallelIterator, ParallelIterator};
+use tokio::{
+    sync::mpsc::{channel, Receiver, Sender},
+    task,
+};
+
+#[derive(Parser)]
+#[command(version)]
+pub struct Cli {
+    #[command(subcommand)]
+    pub command: Command,
+}
+
+#[derive(Subcommand)]
+pub enum Command {
+    SeedStore {
+        #[arg(short, long, value_name = "DUMP_FILE", default_value = "./miden-store.sqlite3")]
+        dump_file: PathBuf,
+
+        #[arg(short, long, value_name = "NUM_ACCOUNTS")]
+        num_accounts: usize,
+
+        #[arg(short, long, value_name = "GENESIS_FILE")]
+        genesis_file: PathBuf,
+    },
+}
+
+const BATCHES_PER_BLOCK: usize = 16;
+const TRANSACTIONS_PER_BATCH: usize = 16;
+const NOTES_PER_TRANSACTION: usize = MAX_OUTPUT_NOTES_PER_BATCH / TRANSACTIONS_PER_BATCH;
+
+/// Create and store blocks into the store. Create a given number of accounts, where each account
+/// consumes a note created from a faucet. The cli accepts the following parameters:
+/// - `dump_file`: Path to the store database file.
+/// - `num_accounts`: Number of accounts to create.
+/// - `genesis_file`: Path to the genesis file of the store.
```

Comment:
nit: I think these could also be doc comments in the `SeedStore` command as well to make Clap show help accordingly